src/main/java/com/stuypulse/robot/util/Limelight.java

```diff
@@ -27,6 +28,8 @@ public class Limelight {
     private final IntegerEntry idEntry;
     private final DoubleArrayEntry blueBotposeEntry;
     private final DoubleArrayEntry redBotposeEntry;
+    private final NetworkTableEntry txEntry;
```

Comment:
I think this should be a double entry. You guys should also have an entry for ty and maybe tv for checking if you have data or not.